hostinit/setup.go

```diff
@@ -462,6 +462,15 @@ func (init *HostInit) expandScript(s string) (string, error) {
 	return script, err
 }
 
+func unprovisionHost(h *host.Host) {
+	if h == nil {
+		return
+	}
+	if err := h.SetUnprovisioned(); err != nil {
+		grip.Errorf("unprovisioning host %s failed: %s", h.Id, err.Error())
+	}
+}
```

Comment:
this should be a method in the hosts packshr

Reply:
moved